README.md

```diff
@@ -116,6 +116,33 @@ For more imformation about the core tests, including what tests are available, h
 * **WooCommerce E2E Environment** - https://www.npmjs.com/package/@woocommerce/e2e-environment
 * **WooCommerce E2E Core Tests** - https://www.npmjs.com/package/@woocommerce/e2e-core-tests
 
+### Examples
+
+This section provides examples of plugins and packages that are implementing WooCommerce related end-to-end tests.
+
+**Woo Manage Fraud Orders**
+
+[Woo Manage Fraud Orders](https://github.com/prasidhda/woo-manage-fraud-orders) is making use of the following packages in their end-to-end tests:
+
+* [**WooCommerce E2E Environment**](https://www.npmjs.com/package/@woocommerce/e2e-environment)
+* [**WooCommerce E2E Utils**](https://www.npmjs.com/package/@woocommerce/e2e-utils)
+* [**WooCommerce API**](https://www.npmjs.com/package/woocommerce-api)
```

Comment:
```suggestion
* [**WooCommerce API**](https://www.npmjs.com/package/@woocommerce/api)
```

Reply:
Good call out, I fixed all instances of this in the README.